schema/proto3/vpc.proto

```diff
@@ -23,16 +23,17 @@ option java_outer_classname = "Vpc";
 
 import "common.proto";
 
-message VpcConfiguration {
+message VpcConfiguration {  // are we ready to remove VpcState based on the current router discussion?
```

Comment:
Is VpcConfiguration in any case used for info purpose?

Reply:
not during our discussion so far including current router design

Reply:
Okay, I could bear with it for some more time unless we complete the design for all major features. It is up to you though :-)

Reply:
I will change the comment to below so that we can bear with it for now :)

// Not used based on the current design including router E2E
// To be removed once we complete the design for all major features
message VpcConfiguration {

Reply:
when would you remove it @er1cthe0ne

Reply:
To be removed once we complete the design for all major features, on top of my head, that would include SG, EIP, SNAT.